torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -578,6 +579,27 @@ std::shared_ptr<FutureMessage> RequestCallbackImpl::processMessage(
          rpc = (std::shared_ptr<RpcCommandBase>)std::move(rpc),
          messageType = request.type(),
          id = request.id()]() {
+          // This cost of pre-request check is minimal thanks to
+          // std::shared_lock. The cost is in magnitude
+          // of 10us. If server global profiler is enabled, we futher pay the
+          // cost of thread local profiler state initialization .
+          // int64_t startTime = ::torch::autograd::profiler::getTime();
+          bool isServerProcessGlobalProfilerEnabled =
```

Comment:
This means profiler will never care about the time spent on waiting for RRef confirmations?

Reply:
Profiler only take into account Tensor "function"s and user specified `RecordFunction` scope. It's not aware of RPC time by default.